sonaranalyzer-dotnet/its/regression-test.ps1

```diff
@@ -36,7 +36,7 @@ function Build-Project([string]$ProjectName, [string]$SolutionRelativePath) {
     Restore-Packages $msbuildVersion $solutionPath
     # Note: Summary doesn't work for MSBuild 14
     Invoke-MSBuild $msbuildVersion $solutionPath `
-        /m `
+        /m:$CpuCount `
```

Comment:
Added a new parameter to limit the parallelism when building AutoMapper, see comment below